.github/CODEOWNERS

```diff
@@ -0,0 +1,135 @@
+
+
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+*       @rrsettgast
+
+/.github/ @rrsettgast
+
+/inputFiles/ @CusiniM @cssherman @jhuang2601
+
+/src/coreComponents/LvArray                           @rrsettgast @wrtobin @corbett5 @CusiniM 
+/src/coreComponents/codingUtilities                   @rrsettgast
+/src/coreComponents/common                            @rrsettgast
+/src/coreComponents/constitutive                      @rrsettgast @CusiniM @dkachuma 
+/src/coreComponents/dataRepository                    @rrsettgast @wrtobin
+/src/coreComponents/denseLinearAlgebra                @rrsettgast
+/src/coreComponents/discretizationMethods             @rrsettgast
+/src/coreComponents/events                            @rrsettgast @cssherman 
+/src/coreComponents/fieldSpecification                @rrsettgast
+/src/coreComponents/fileIO                            @rrsettgast @wrtobin
+/src/coreComponents/finiteElement                     @rrsettgast
+/src/coreComponents/finiteVolume                      @rrsettgast
+/src/coreComponents/functions                         @rrsettgast
+/src/coreComponents/linearAlgebra                     @rrsettgast @castelletto1 @victorapm 
+/src/coreComponents/mainInterface                     @rrsettgast
+/src/coreComponents/math                              @rrsettgast
+/src/coreComponents/mesh                              @rrsettgast @wrtobin
+/src/coreComponents/physicsSolvers                    @rrsettgast
+/src/coreComponents/physicsSolvers/contact            @CusiniM
+/src/coreComponents/physicsSolvers/fluidFlow          @CusiniM @paveltomin 
+/src/coreComponents/physicsSolvers/inducedSeismicity  @CusiniM
+/src/coreComponents/physicsSolvers/multiphysics       @CusiniM @paveltomin @castelletto1 
+/src/coreComponents/physicsSolvers/python             @cssherman
+/src/coreComponents/physicsSolvers/simplePDE          
+/src/coreComponents/physicsSolvers/solidMechanics     @rrsettgast @castelletto1 @CusiniM 
+/src/coreComponents/physicsSolvers/surfaceGeneration  @rrsettgast @CusiniM @jhuang2601
+/src/coreComponents/physicsSolvers/wavePropagation    @rrsettgast @sframba
+/src/coreComponents/schema                            @rrsettgast
+/src/coreComponents/unitTests                         @rrsettgast
```

Comment:
actually, can we exclude `/src/coreComponents/schema `? It's autogenerated and every time someone modifies even just a wrapper description it will be modified. So basically the codeowners of this folder will basically have to approve 95% of the PRs.

Reply:
@CusiniM @rrsettgast 
And about `/src/coreComponents/dataRepository`? Is it possible to add me later on it (as I may contribute quite extensively on this folder in the future).